frontend/src/components/render-page/markdown-document.tsx

```diff
@@ -24,6 +24,7 @@ export interface MarkdownDocumentProps extends RendererProps {
   additionalRendererClasses?: string
   disableToc?: boolean
   baseUrl: string
+  newLinesAreBreaks?: boolean
```

Comment:
Am i wrong this will be a required type. As this will always be either true or false?